erpnext/education/doctype/course_schedule/course_schedule_calendar.js

```diff
@@ -8,6 +8,7 @@ frappe.views.calendar["Course Schedule"] = {
 		"allDay": "allDay"
 	},
 	gantt: false,
+	order_by: "from_time",
```

Comment:
Shouldn't be `schedule_date` handier for the user? otherwise irrespective of different dates all the course schedule will come on the same day since from_time is a Time field.